### ISSUE TITLE
v2.x branch fails to build with released plfs available

### ISSUE THREAD

**Issue by hppritcha** [open-mpi/ompi#1271]
issue reported by @loveshack, but filed on open-mpi/ompi-release#880

The build will fail if you have the released version (2.5) of plfs installed because the signature of plfs_setxattr used by OMPI's fs component was changed after the release.  I think configure should check for the correct signature of plfs_setxattr.

Closing the issue opened on ompi-release as a dup of this one.

**Comment by edgargabriel** [comment]
Will look into this next week.

**Comment by edgargabriel** [comment]
I have a fix for that in my private repository. I went through the plfs releases, and I noticed that all released versions of plfs had the three argument version, not the four (we must have had a pre-release version when we wrote the code), so I don't think that we need to add configure logic, I just changed our own code, and it compiles now correctly.

That being said, since I haven't looked into the plfs components in a long time, I used this opportunity to run some more tests and I noticed a number of other items. I am working on some additional fixes, and will file a larger pr containing all the fixes for the plfs file system,. Will take me however a day or two.
